### PR TITLE
release 1.0

### PR DESCRIPTION
**Pull request by zhu733756**
Signed-off-by: zhu733756 <zhu733756@kubesphere.io>

- update Changelog.md && Release.md && VERION

### CODE REVIEW

**Comment by zhu733756** [comment]
I added `release-1.0` branch, PTAL. /cc @benjaminhuo @wanjunlei @wenchajun

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+
+- [FEATURE] Renaming fluentbit-operator to fluent-operator #189 #190
+- [FEATURE] Add more fluentd examples for fluent-operator #194
+- [FEATURE] Integrate Fluentd to helm charts #204 #208 
+- [FEATURE] Encrypt sensitive information for output plugin #219 
+- [FEATURE] Multi-workers in a single fluentd pod #194
+- [FEATURE] Integrate e2e/function tests for generating fluentd configuration. #203 #206 
+- [FEATURE] Enhance docs explanations #199 #228
+- [FEATURE] Refactor multi images/binaries build, add github CI. #213 #214 
+- [ENHANCEMENT] Fix ci and make the repository importable and downloadable #229
```
Comment (benjaminhuo) [inline CHANGELOG.md:11]: need to add changes since v0.12.0 is released

Comment (zhu733756) [reply CHANGELOG.md:21]: fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,28 @@
...
 - [FEATURE] Encrypt sensitive information for output plugin #219 
 - [FEATURE] Multi-workers in a single fluentd pod #194
 - [FEATURE] Integrate e2e/function tests for generating fluentd configuration. #203 #206 
 - [FEATURE] Enhance docs explanations #199 #228
-- [FEATURE] Refactor multi images/binaries build, add github CI. #213 #214 
+- [FEATURE] Refactor multi images/binaries build, add github CI. #152 #154 #213 #214
+- [ENHANCEMENT] Set the crictl path to a variable (#181) 
 - [ENHANCEMENT] Fix ci and make the repository importable and downloadable #229
 - [ENHANCEMENT] Fix CR status patching #225
+- [ENHANCEMENT] Improved kafka plugin (#182)
```
Comment (benjaminhuo) [inline CHANGELOG.md:21]: Improved kafka plugin
=> 
Add more fields to the Kafka plugin

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,15 +1,28 @@
...
 - [FEATURE] Integrate Fluentd to helm charts #204 #208 
 - [FEATURE] Encrypt sensitive information for output plugin #219 
 - [FEATURE] Multi-workers in a single fluentd pod #194
 - [FEATURE] Integrate e2e/function tests for generating fluentd configuration. #203 #206 
 - [FEATURE] Enhance docs explanations #199 #228
-- [FEATURE] Refactor multi images/binaries build, add github CI. #213 #214 
+- [FEATURE] Refactor multi images/binaries build, add github CI. #152 #154 #213 #214
+- [ENHANCEMENT] Set the crictl path to a variable (#181) 
 - [ENHANCEMENT] Fix ci and make the repository importable and downloadable #229
 - [ENHANCEMENT] Fix CR status patching #225
```
Comment (benjaminhuo) [inline CHANGELOG.md]: Fix CR status patching => Refine Fluentd CRs status

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,15 +1,28 @@
...
+- [FEATURE] Add multiline filter plugin (#176) 
+- [FEATURE] Firehose plugin support (#178)
+- [FEATURE] Add volume crd (#186)
 - [FEATURE] Renaming fluentbit-operator to fluent-operator #189 #190
 - [FEATURE] Add more fluentd examples for fluent-operator #194
 - [FEATURE] Integrate Fluentd to helm charts #204 #208 
 - [FEATURE] Encrypt sensitive information for output plugin #219 
 - [FEATURE] Multi-workers in a single fluentd pod #194
 - [FEATURE] Integrate e2e/function tests for generating fluentd configuration. #203 #206 
 - [FEATURE] Enhance docs explanations #199 #228
```
Comment (benjaminhuo) [inline CHANGELOG.md]: Enhance docs explanations => Refine docs

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,15 +1,28 @@
...
+- [FEATURE] Multiline support (#172)
+- [FEATURE] Add aws filter plugin (#173)
+- [FEATURE] Add multiline filter plugin (#176) 
+- [FEATURE] Firehose plugin support (#178)
+- [FEATURE] Add volume crd (#186)
 - [FEATURE] Renaming fluentbit-operator to fluent-operator #189 #190
 - [FEATURE] Add more fluentd examples for fluent-operator #194
 - [FEATURE] Integrate Fluentd to helm charts #204 #208 
 - [FEATURE] Encrypt sensitive information for output plugin #219 
 - [FEATURE] Multi-workers in a single fluentd pod #194
```
Comment (benjaminhuo) [inline CHANGELOG.md]: Enable multi-workers in one Fluentd pod

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,15 +1,28 @@
...
+- [FEATURE] feature: add datadog_output type (#149)  
+- [FEATURE] Add support for rewrite tag (#155)  
+- [FEATURE] Multiline support (#172)
+- [FEATURE] Add aws filter plugin (#173)
+- [FEATURE] Add multiline filter plugin (#176) 
+- [FEATURE] Firehose plugin support (#178)
+- [FEATURE] Add volume crd (#186)
 - [FEATURE] Renaming fluentbit-operator to fluent-operator #189 #190
 - [FEATURE] Add more fluentd examples for fluent-operator #194
 - [FEATURE] Integrate Fluentd to helm charts #204 #208
```
Comment (benjaminhuo) [inline CHANGELOG.md]: => Add Fluentd to helm charts

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,15 +1,28 @@
...
+- [FEATURE] Add support for RetryLimit in outputs (#148) 
+- [FEATURE] feature: add datadog_output type (#149)  
+- [FEATURE] Add support for rewrite tag (#155)  
+- [FEATURE] Multiline support (#172)
+- [FEATURE] Add aws filter plugin (#173)
+- [FEATURE] Add multiline filter plugin (#176) 
+- [FEATURE] Firehose plugin support (#178)
+- [FEATURE] Add volume crd (#186)
 - [FEATURE] Renaming fluentbit-operator to fluent-operator #189 #190
 - [FEATURE] Add more fluentd examples for fluent-operator #194
```
Comment (benjaminhuo) [inline CHANGELOG.md]: Add more Fluentd examples

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,15 +1,28 @@
 ## 1.0.0 / 2022-03-11
-
+- [FEATURE] Add priority class to FluentBit type (#146) 
+- [FEATURE] Add support for RetryLimit in outputs (#148) 
+- [FEATURE] feature: add datadog_output type (#149)  
+- [FEATURE] Add support for rewrite tag (#155)  
+- [FEATURE] Multiline support (#172)
+- [FEATURE] Add aws filter plugin (#173)
+- [FEATURE] Add multiline filter plugin (#176) 
+- [FEATURE] Firehose plugin support (#178)
```
Comment (benjaminhuo) [inline CHANGELOG.md:10]: Add Fluent Bit Firehose plugin support

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,15 +1,28 @@
 ## 1.0.0 / 2022-03-11
-
+- [FEATURE] Add priority class to FluentBit type (#146) 
+- [FEATURE] Add support for RetryLimit in outputs (#148) 
+- [FEATURE] feature: add datadog_output type (#149)  
+- [FEATURE] Add support for rewrite tag (#155)  
+- [FEATURE] Multiline support (#172)
```
Comment (benjaminhuo) [inline CHANGELOG.md:7]: Add Fluent Bit multiline logs support

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,15 +1,28 @@
 ## 1.0.0 / 2022-03-11
-
+- [FEATURE] Add priority class to FluentBit type (#146) 
+- [FEATURE] Add support for RetryLimit in outputs (#148) 
+- [FEATURE] feature: add datadog_output type (#149)
```
Comment (benjaminhuo) [inline CHANGELOG.md]: feature: add datadog_output type
=>
Add Fluent Bit Datadog output

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: tests/scripts/fluentd_e2e.sh [hunk 11]
```diff
@@ -30,8 +31,8 @@ function build_image() {
 }
 
 function start_fluent_operator() {
-  cd $PROJECT_ROOT 
-  cat manifests/setup/setup.yaml | sed  "s#kubesphere/fluent-operator:latest#kubesphere/fluent-operator:${IMAGE_TAG}#g"  | kubectl apply -f -
+  cd $PROJECT_ROOT
```
Comment (benjaminhuo) [inline tests/scripts/fluentd_e2e.sh:34]: file https://github.com/fluent/fluent-operator/blob/master/VERSION should be changed too

File: VERSION [hunk 12]
```diff
@@ -1 +1 @@
-latest
+v1.0.0
```
Comment (zhu733756) [inline VERSION:1]: Changed here. The master branch no need to change.

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,3 +1,30 @@
+## 1.0.0 / 2022-03-11
+
```
Comment (benjaminhuo) [inline CHANGELOG.md:2]: Need to add a breaking change section here like https://github.com/kubernetes-sigs/kubebuilder/releases

### Breaking Changes:
- All Fluent Bit CRDs has been changed from namespaced scope to cluster scope
- Fluentd CRDs has been added

### Features
- xx

### Enhancements

### Fixes

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,3 +1,39 @@
...
+### Breaking Changes:
+- All Fluent Bit CRDs has been changed from namespaced scope to cluster scope
+- Fluentd CRDs and Fluentd Operator has been added
```
Comment (benjaminhuo) [inline CHANGELOG.md:5]: CRDs and controller for Fluentd have been added

File: CHANGELOG.md [hunk 15]
```diff
@@ -1,3 +1,39 @@
+## 1.0.0 / 2022-03-11
+
+### Breaking Changes:
```
Comment (benjaminhuo) [inline CHANGELOG.md:3]: Remove : from ### Breaking Changes:

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**